index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>videos of kids songs</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <h1>YOUTUBE</h1>
+    <p>below i have put together five different short videos about kids songs....</p>
+    
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/1HEF8kuu7t0" 
+    title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
+     clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+     <!--to do navigaton bars which is a list of unorderd list --> <hr><br>
+     <ul>
+         <li class="active"><a href="share.asp">Share</a></li>
+         <li class="active"><a href="download.asp">Download</a></li>
+     </ul>
+     <!--to creat a text feild,we use the input element and they are self closing.input element appears 
+        as a box that permits users to write in it  .--><br>
+        <input type="text" placeholder="search here">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/ETfLstwcZxY" 
+        title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;q3vHw" 
+         title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
+         clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><hr><br>
+         <iframe width="560" height="315" src="https://www.youtube.com/embed/tFoUuF
+          clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><hr><br>
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/ThAQjOJT5iU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; 
+          clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><hr><br>
+          
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/pR_LON6fJy8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
+           clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><hr><br>
```

Comment:
good job but please i suggest you try to apply flex on the elements and also give them padding and margin